stable/rabbitmq/values.yaml

```diff
@@ -71,8 +71,11 @@ rabbitmq:
   ulimitNofiles: '65536'
 
   ## Plugins to enable
-  plugins: |-
-      [rabbitmq_management, rabbitmq_peer_discovery_k8s].
+  plugins: "rabbitmq_management rabbitmq_peer_discovery_k8s"
+
+  ## Extra plugins to enable
+  ## Use this instead of `plugins` to add new plugins
+  # extraPlugins: ""
```

Comment:
is the empty string a valid value when constructing the erlang config value? 
if not, maybe show a valid value for commented-out `extraPlugins` here.
if yes, maybe explicitly define it as empty string instead of implicit `nil`.

Reply:
How will the **extraPlugins** values look in this case?
eg. **extraPlugins**: "rabbitmq_shovel rabbitmq_shovel_management" or 
eg  **extraPlugins**: |
rabbitmq_shovel, 
rabbitmq_shovel_management"